cmd/cloudx/relationtuples/relationtuples.go

```diff
@@ -0,0 +1,126 @@
+package relationtuples
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+
+	ketoClient "github.com/ory/keto/cmd/client"
+	"github.com/ory/keto/cmd/relationtuple"
+	"github.com/ory/x/randx"
+
+	"github.com/ory/cli/cmd/cloudx/client"
+)
+
+const FlagAll = "all"
+
+var ErrDeleteMissingAllFlag = fmt.Errorf("you must either select tuples to delete, or use `--all' to delete all tuples")
```

Comment:
Suggestion for friendlier error message:

```suggestion
var ErrDeleteMissingAllFlag = fmt.Errorf("please select the tuples to delete or use `--all` to delete all tuples")
```

Reply:
Rename to `ErrMissingTuplesToDelete` to make the this error more self-describing?